smssync/src/main/java/org/addhen/smssync/presentation/task/state/State.java

```diff
@@ -52,7 +52,7 @@ public boolean isInitialState() {
     }
 
     public boolean isRunning() {
-        return EnumSet.of(SyncState.SYNC).contains(state);
+        return state == SyncState.SYNC;
```

Comment:
Though this works I prefer the `SyncState.Sync.equals(state);` as it reads much better to me. You using `==` for null safety?